core/meterer/util.go

```diff
@@ -0,0 +1,164 @@
+package meterer
+
+import (
+	"context"
+	"math/big"
+
+	commonaws "github.com/Layr-Labs/eigenda/common/aws"
+	test_utils "github.com/Layr-Labs/eigenda/common/aws/dynamodb/utils"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+
+func DummyCommitment() core.G1Point {
+	commitment := core.NewG1Point(big.NewInt(123), big.NewInt(456))
+	return *commitment
+}
+
+func CreateReservationTable(clientConfig commonaws.ClientConfig, tableName string) error {
```

Comment:
I feel like we can collapse these tables into a single table design. Wdyt?

Reply:
🤔 something like [this](https://gist.github.com/hopeyen/7fa8f7c38b9d9a9b78beadf4c818c57e)?
My main consideration is the read/write performance based on the evenness of key distributions, and think it is clearer if different payment methods live in different tables, but I'm interested in learning the advantages of combining these into a single table

Reply:
Added a suggestion in the design doc. We can discuss it there